Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -267,6 +267,10 @@ class CompileWebKit(shell.Compile):
     description = ["compiling"]
     descriptionDone = ["compiled"]
     warningPattern = ".*arning: .*"
+    
+    def __init__(self, timeout=None, **kwargs):
+        timeout = timeout or 60*30
+        super().__init__(timeout, **kwargs)
```

Comment:
can change to something like:
`super(CompileWebKit, self).__init__(timeout=30*60, **kwargs)`

Reply:
You can't pass the same parameter twice, so this would break cases like https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/build-webkit-org/steps.py#L157, where a caller provides their own timeout.